managed/src/main/java/com/yugabyte/yw/models/YugawareProperty.java

```diff
@@ -24,7 +26,7 @@
   public static final Logger LOG = LoggerFactory.getLogger(YugawareProperty.class);
 
   // The name of the property.
-  @Id private String name;
+  @Id public String name;
```

Comment:
Why public?

Reply:
not sure how it got changes, let me change.